Marlin/stepper.cpp

```diff
@@ -959,51 +969,51 @@ void st_init()
 
   //Initialize Step Pins
   #if defined(X_STEP_PIN) && (X_STEP_PIN > -1)
-    SET_OUTPUT(X_STEP_PIN);
-    WRITE(X_STEP_PIN,INVERT_X_STEP_PIN);
+    X_STEP_INIT;
+    X_STEP_WRITE(INVERT_X_STEP_PIN);
     disable_x();
```

Comment:
A new macro called `OUT_WRITE(p,v)` combines `SET_OUTPUT(p)` with `WRITE(p,v)`. If you do an upstream merge from Development you might get a conflict on these lines. When you resolve the conflict, your new code here should supersede the lines that have `OUT_WRITE` in them now. If you feel like following the same single-macro pattern you could add macros like...

    #define X_STEP_INIT_WRITE(v) OUT_WRITE(X_STEP_PIN,v)